RELEASES.md

```diff
@@ -1,3 +1,119 @@
+Version 1.39.0 (2019-11-07)
+===========================
+
+Language
+--------
+- [You can now create `async` functions and blocks with `async fn` and
+  `async {}` respectively, and you can now call `.await` on async expressions.][63209]
+- [You can now use `cfg` attributes on function parameters.][64010] e.g.
+  ```rust
+  fn len(
+      #[cfg(windows)] slice: &[u16],
+      #[cfg(not(windows))] slice: &[u8],
+  ) -> usize {
+      slice.len()
+  }
+  ```
+- [You can now bind patterns by move and bind by reference in `if` guards.][63118]
+  e.g.
+  ```rust
+  fn main() {
+      let array: Box<[u8; 4]> = Box::new([1, 2, 3, 4]);
+
+      match array {
+          nums
+  //      ---- `nums` is bound by move.
+              if nums.iter().sum::<u8>() == 10
+  //                 ^------ `.iter()` implicitly takes a reference to `nums`.
+          => {
+              drop(nums);
+  //          ----------- Legal as `nums` was bound by move and so we have ownership.
+          }
+          _ => unreachable!(),
+      }
+  }
+  ```
+
+
+
+Compiler
+--------
+- [Added tier 3\* support for the `i686-unknown-uefi` target.][64334]
+- [Added tier 3 support for the `sparc64-unknown-openbsd` target.][63595]
+- [rustc will now trim code snippets in diagnostics to fit in your terminal.][63402]
```

Comment:
This does not apply when used with cargo, due to the way the two interact (see https://github.com/rust-lang/cargo/pull/7315).  I'm not sure if it should be called out if it won't work for most users who use cargo to build.

Reply:
It should be pointed out yes since the compiler is an independent program. However, it is a good idea to point out what you just did in the text.